io.catenax.mp_standard_response/1.0.0/ProvideProductionForecast.ttl

```diff
@@ -0,0 +1,166 @@
+   samm:properties ( :positionId :productionForecast :precisionOfForecast :prodcutionStatus [ samm:property :reasons4Delay; samm:optional true ] :returnCode :forecastDate ) .
+
+:positionId a samm:Property ;
+   samm:preferredName "positionId"@en ;
+   samm:description "Identifier of a position of an order"@en ;
+   samm:characteristic :String ;
+   samm:exampleValue "00000000-0000-0000-C000-000000000046" .
+
+:productionForecast a samm:Property ;
+   samm:preferredName "productionForecast"@en ;
+   samm:description "date of completion"@en ;
+   samm:characteristic :Datetime .
+
+:precisionOfForecast a samm:Property ;
+   samm:preferredName "precisionOfForecast"@en ;
+   samm:description "accuracy of the prediction"@en ;
+   samm:characteristic :TimeValueCharacteristic .
+
+:prodcutionStatus a samm:Property ;
+   samm:preferredName "prodcutionStatus"@en ;
+   samm:description "Status of the order/position within MP"@en ;
+   samm:characteristic :ProductionStatusEnum ;
+   samm:exampleValue "ItemPlanned" .
+
+:reasons4Delay a samm:Property ;
+   samm:preferredName "reasons4Delay"@en ;
+   samm:description "optional field to provide information to the customer why a delivery date is not met"@en ;
+   samm:characteristic :Reasons4DelayEnum ;
+   samm:exampleValue "supply problems" .
+
+:returnCode a samm:Property ;
+   samm:preferredName "returnCode"@en ;
+   samm:description "return code that indicates whether a single item of an order matches the customers desired request"@en ;
+   samm:characteristic :ReturnCodeEnum ;
+   samm:exampleValue "lower accuracy of precision" .
+
+:forecastDate a samm:Property ;
+   samm:preferredName "forecastDate"@en ;
+   samm:description "Date/time of the forecast calculation"@en ;
+   samm:characteristic :Datetime .
+
+:String a samm:Characteristic ;
+   samm:preferredName "String"@en ;
+   samm:description "String values"@en ;
+   samm:dataType xsd:string .
+
+:Datetime a samm:Characteristic ;
+   samm:preferredName "Datetime"@en ;
+   samm:description "date of completion of an order position"@en ;
+   samm:dataType xsd:dateTime .
+
+:TimeValueCharacteristic a samm:Characteristic ;
+   samm:preferredName "TimeValueCharacteristic"@en ;
+   samm:description "Link to the  TimeUnit Data Type"@en ;
+   samm:dataType ext-unsubscribe:TimeValue .
+
+:ProductionStatusEnum a samm-c:Enumeration ;
+   samm:preferredName "ProductionStatusEnum"@en ;
+   samm:description "Enum with possible states of an order within MP"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "itemReceived" "itemPlanned" "itemInProduction" "itemCompleted" "statusUndefined" ) .
+
+:Reasons4DelayEnum a samm-c:Enumeration ;
+   samm:preferredName "Reasons4DelayEnum"@en ;
+   samm:description "Enum that specifies reasons for a delay of an order"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "supply problems" "internal problems" "other circumstances" "no Information available" ) .
```

Comment:
all but the I in Information lower case? Either write all upper case or turn I to lowercase